.github/pull_request_template.md

```diff
@@ -1 +1,5 @@
-**Before submitting your PR:** Please read through the contribution guide at https://connectrpc.com/connect/blob/main/.github/CONTRIBUTING.md
+<!--
+Before submitting your PR, please read through the contribution guide!
+
+https://connectrpc.com/connect/blob/main/.github/CONTRIBUTING.md
```

Comment:
The link leads to a 404 for me. 

Should we use https://github.com/connectrpc/connect-go/blob/main/.github/CONTRIBUTING.md instead?

Reply:
```suggestion
https://github.com/connectrpc/connect-go/blob/main/.github/CONTRIBUTING.md
```